dell-inspiron-7559-linux-guide/index.html

```diff
@@ -1,258 +1,440 @@
-                                border: 1px solid #ffa0a0;
-                        }
+                .note {
+                        display: block;
+                        padding: 10px;
+                }
+                
+                .note-18 {
+                        background: #90CAF9;
+                        border: 1px solid #90CAF9;
+                }
 
-                        .note-info {
-                                background: #ceffc4;
-                                border: 1px solid #acff9b;
-                        }
+                .note-warn {
+                        background: #ffbaba;
+                        border: 1px solid #ffa0a0;
+                }
 
-                </style>
-        </head>
+                .note-info {
+                        background: #ceffc4;
+                        border: 1px solid #acff9b;
+                }
+        </style>
+</head>
 
-        <body>
-                <div id="main-content">
-                        <h1><a name="top-title">Dell Inspiron 7559 Ubuntu Linux Guide</a></h1>
-                        <p>Last updated <b>9 Nov 2017</b></p>
-                        <p class="note note-warn">I no longer own this laptop and cannot continue to update this guide. If someone would like to fork this on Github or something and actively maintain it, then I will point to yours.</p>
-                        <p>This guide will show you how to install and configure an <strong>Ubuntu Linux 16.04 LTS</strong> distribution (Ubuntu, Lubuntu, Xubuntu, Kubuntu, Ubuntu Gnome, Ubuntu MATE) on the Dell Inspiron 7559 laptop.</p>
-                        <p class="note note-warn"><strong>DISCLAIMER</strong> This document just details the processes that I discovered through hours of trial and error to get a stable Ubuntu distribution working on my laptop. <em>I'm not liable for any damage or adverse effects that this has on your system.</em> Always back up your important data before doing any major work to your computer.</p>
+<body>
+        <div id="main-content">
+                <h1>
+                        <a name="top-title">Dell Inspiron 7559 Ubuntu Linux Guide</a>
+                </h1>
+                <p>Last updated
+                        <b>10 July 2018</b>
+                </p>
+                <p>This guide will show you how to install and configure an
+                        <strong>Ubuntu Linux 16.04/18.04 LTS</strong> distribution (Ubuntu, Lubuntu, Xubuntu, Kubuntu, Ubuntu Gnome, Ubuntu
+                        MATE) on the Dell Inspiron 7559 laptop.</p>
+                <p class="note note-warn">Donot try to upgrade from 
+                        <b>Ubuntu 16.04 LTS</b> to <b>18.04 LTS now</b>. This might crash your OS (Crashed mine! Had to do a fresh install of
+                        Ubuntu 18.04 LTS.). Upgrade from 16.04 LTS to 18.04 LTS will be enabled 
+                <a href="https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes">late july</a>.
```

Comment:
I'd suggest linking directly to the relevant section: [https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes#Upgrading_from_Ubuntu_16.04_LTS](https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes#Upgrading_from_Ubuntu_16.04_LTS)